src/app/grid/grid-filtering-template-sample/grid-filtering-template-sample.component.ts

```diff
@@ -76,14 +76,12 @@ export class FilteringTemplateSampleComponent implements OnInit {
     }
 
     public onDateSelected(event, column: IgxColumnComponent) {
```

Comment:
This code needs to be changed as follows in order for the clear button to work and actually clear the filter:
```
if (!event) {
            this.clearInput(column);
            return;
        }

        this._filterValues.set(column, event);
        this.grid1.filter(column.field, event, IgxDateFilteringOperand.instance().condition('equals'),
                column.filteringIgnoreCase);
```